dataland-frontend/src/components/pages/CompanyInformation.vue

```diff
@@ -96,7 +99,7 @@ export default defineComponent({
     },
     contextMenuItems() {
       const listOfItems = [];
-      if (!this.isUserDataOwner && this.userId) {
+      if (!this.isUserDataOwner && this.userId !== undefined) {
```

Comment:
In main it will check on authenticated instead of the userId

Reply:
yes - better way

Reply:
ok